src/Symfony/Component/Serializer/NameConverter/NameConverterInterface.php

```diff
@@ -20,11 +20,19 @@ interface NameConverterInterface
 {
     /**
      * Converts a property name to its normalized value.
+     *
+     * @param class-string|null $class
+     * @param string|null $format
+     * @param array<string, mixed> $context
```

Comment:
Let align them all (below) also and add this to the implemetations above:
```suggestion
     * @param class-string|null    $class
     * @param string|null          $format
     * @param array<string, mixed> $context
```